README.md

```diff
@@ -91,11 +95,11 @@ AWS.IAM.MFAEnabled
 [INFO]: <current working directory>/tmp/panther-analysis-2020-03-23T12-48-18.zip
 ```
 
-### Upload packages to Panther directly. 
+Or upload packages directly into Panther:
 
 _Note, this expects your environment to be setup the same way as if you were using the AWS CLI, see the setup instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html). We also recommend using a credentials manager such as [aws-vault](https://github.com/99designs/aws-vault)._
```

Comment:
Is this still true? Can't we use API keys OR AWS creds?

Reply:
correct users can utilize an api key OR aws creds, probably be good to call that out I don't think we ever updated readme to reflect that

Reply:
Thanks @darwayne can you stamp again? I removed that line.